.github/workflows/build-monitoring-containers.yml

```diff
@@ -0,0 +1,56 @@
+name: Create and publish a Docker images
+
+on:
+  workflow_dispatch:
+
+  push:
+    branches: ['main', 'master']
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for grafana container
+        id: grafana-meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: ${{ env.REGISTRY }}/emqx/perf-test-grafana
```

Comment:
Curiosity: what happens if the image does not exist (like the first time creating)?

Reply:
It needs to be build by the CI, which wasn't set up at the time of writing the PR

Reply:
Right!  I was just thinking it might return empty tags and labels during the first execution by CI, and that it maybe would need to set default values to bootstrap it.